src/components/DataPlatformProviderPanel.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ArrowTopRightOnSquareIcon, XMarkIcon } from '@heroicons/react/24/solid';
+import {
+  Panel,
+  PanelActions,
+  PanelBody,
+  PanelHeader,
+  PanelTitle,
+  PanelTitleTags,
+  PanelTitleWrapper,
+} from './Panel';
+import { OffsiteLink } from './OffsiteLink';
+import { DataPlatformProviderTable } from './DataPlatformProviderTable';
+
+interface DataPlatformProviderLinkProps {
+  provider: string;
+  applicantId: string | undefined;
+}
+
+/**
+   * Format a tax ID in the standardized US EIN format (##-#######).
+   *
+   * @param  {string} taxId
+   * @return {string} The tax ID, formatted as ##-#######
+   */
+const formatAsEin = (taxId: string) => taxId
+  .replace(/[^0-9]/g, '') // Reduce to numbers only
+  .substring(0, 9) // Reduce to nine characters
+  .replace(/(\d{2})(\d{7})/, '$1-$2'); // Format as ##-#######
+
+const ProviderLinkPanelTag = ({
+  provider,
+  applicantId = '',
+}: DataPlatformProviderLinkProps) => {
+  let providerName = '';
+  let providerUrl = '';
+
+  switch (provider) {
+    case 'candid':
+      providerName = 'Candid';
+      providerUrl = `https://www.guidestar.org/profile/${formatAsEin(applicantId)}`;
```

Comment:
Candid does provide a URL to link to in its response! I don't believe Charity Navigator does, though, so we'll still need to compose a link to that one.